Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -340,7 +341,11 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor {
    * This will cause more tiles to be loaded, but helps to avoid holes and
    * provides a more consistent mesh, which may be helpful for physics.
    */
-  UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Cesium|Tile Culling")
+  UPROPERTY(
+      EditAnywhere,
+      BlueprintReadWrite,
+      Category = "Cesium|Tile Culling",
+      Meta = (EditCondition = "!UseLodTransitions"))
   bool EnableFrustumCulling = true;
```

Comment:
Is there any way to ensure that this gets unchecked in the Details panel whenever UseLodTransitions is on? Otherwise, this gets confusing because the setting is uneditable, and may be on, but under the hood it's actually off. If we can't tie it to the actual used frustrum culling setting, we may want to add EditConditionHides.